classes_assignment.js

```diff
@@ -0,0 +1,58 @@
+// 1.
+//   * Create a Human class that takes in a name and age.
+//   * Add the function `ageOneYear` that ages the human.
+//   * Add the function `eating`, that logs "mmm, mmm, mmm, I'm love'n it".
+//   * Create an instance of the Human class.
+//     * console log your humans age
+//     * call ageOneYear on your human
+//     * console log their age again.
+//     * call eating on your human.
+//
+class Human {
+  constructor(name, age){
+    this.name = name,
+    this.age = age
+  }
+
+  ageOneYear(num) {
+    return this.age++;
+  }
+
+  eating(str){
+    return `${Human.name}mmm, mmm, mmmm, I'm love'n it"`
```

Comment:
Just to add to what Michelle said, when a question calls for you to log something, it means to console.log the result. Otherwise, you will just return the result.